### DISCUSSION TITLE
GitHub Awards 2023 Announced 🌟

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#74929] [Announcements]
We've got some electrifying news to share—brace yourselves for the grand reveal of our GitHub Awards 2023! 🌟 This is no ordinary announcement; it's a celebration of YOU and your incredible contributions to the developer community.

We announced the award winners at [GitHub Universe 2023](http://githubuniverse.com/), but check out the [blog post](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients) for a recap for those that missed it or want to relive the excitement!

Here are the categories below:

# Open Source Awards

## Wonderfully Welcoming Award
_The Wonderfully Welcoming Award recognizes people or projects that have been the most welcoming and has seen an increasing amount of contributors. **Congratulations to the [Python Software Foundation](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Noteworthy Newcomer Award
_The Noteworthy Newcomer Award recognizes people or projects that are creating and sharing breakthrough contributions/projects. **Congratulations to [LangChain](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Global Grandiose Award
_The Global Grandiose Award recognizes projects that have a large global community of contributors. **Congratulations to [Storybook](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Hardware Hacker Award
_The Hardware Hacker Award recognizes people or projects contributing to making hardware accessible through open specs/software. **Congratulations to [Limor "Ladyada" Fried (@LadyADA)](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Supply Chain Sentinel Award
_The Supply Chain Sentinel Award recognizes a community member part of an advisory who has contributed to make the software supply chain more secure, by finding, disclosing, or fixing security vulnerabilities in open source projects or in supply chain tooling, or by providing information on known vulnerabilities to the ecosystem. **Congratulations to [Alexander Brandes (@notmyfault)](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Audience Choice Award
_The Audience Choice Award recognizes a project that has been voted by developers as a project that has helped them create an impact. **[Check out the Winner here!](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)**_

Did you vote when you registered for GitHub Universe? Then your vote counted 🎉

# Education Awards
## Lighthouse Award
_The Lighthouse Award recognizes students who lead fellow learners by shining a light to their communities through open source. **Congratulations to [Jaisal Patel (@GhostOf0days)](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

Did you know that their contributions in this community were highlighted? 🌟

## Empowering Educator Award
_The Empowering Education Award recognizes educators and teachers who provide support and answers to fellow members of their communities. **Congratulations to [Catherine Leung (@catherine-leung)](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Phenomenal Education Partner Award
_The Phenomenal Education Partner Award recognizes GitHub Education partners that provide strong support and opportunities for students, teachers, and schools. **Congratulations to [Appwrite](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

# Customer Awards
## OSPO Leadership Award
_The OSPO Leadership Award recognizes Open Source Program Offices (OSPOs) who have demonstrated that it's not only possible to innovate in the enterprise by focusing on open source, it's essential. **Congratulations to [Mercedes Benz](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## DevOps Champion Award
_The DevOps Champion Award recognizes GitHub’s brightest customer champions who have shared their best-in-class stories and thought leadership. This award spotlights how they have transformed their developer organizations, enabling our community to learn from their insights and apply these learnings back to their own GitHub journey. **Congratulations to [Katie Peters](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## GitHub for Good Award
The GitHub for Good Award recognizes a nonprofit or social sector organization who has leveraged GitHub for good to empower developers to make a positive difference in the world. **Congratulations to [Ersilia](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)**!

# GitHub Stars Awards
## Star of the Year Award
_The GitHub Star of the Year Award recognizes the GitHub Star that has made a huge impact to the developer community—from growing developer communities from the ground up, helping users get started in open source, and being incredibly welcoming and supporting the community. **Congratulations to [Emanuele Bartolesi (@kasuken)](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Top Teacher Award
_The Top Teacher Award recognizes the Star who thrives on educating, inspiring, and helping everyone grow and nurturing communities. **Congratulations to [Barbara Forbes (@Ba4bes)](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Community Growth Award
_The Community Growth Award recognizes the GitHub Star who grows and nurtures communities around them, whether it’s an OSS project or a developer community, and brings everyone together. **Congratulations to [Miguel Ángel Durán García (@midudev)](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

## Open Source Advocacy Award
_The Open Source Advocacy Award recognizes the GitHub Star that has helped users get started and built up communities using open source. **Congratulations to [Ruth Ikegah (@Ruth-ikegah)](https://github.blog/2023-11-09-celebrating-the-github-awards-2023-recipients)!**_

We believe in recognizing and applauding the outstanding efforts of our community members. Tell your fellow community members, rally your friends, and let's make some noise! 

Share this post, tag your favorite contributors in the comments below, and let them know their efforts are acknowledged and celebrated! 💖👇

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
